src/rigid_filament.cpp

```diff
@@ -295,10 +300,19 @@ void RigidFilament::UpdateSitePositions() {
 void RigidFilament::ApplyForcesTorques() {
   const double *force = bonds_.back().GetForce();
   const double *torque = bonds_.back().GetTorque();
-  for (int i = 0; i < 3; ++i) {
-    force_[i] = force[i];
-    torque_[i] = torque[i];
-  }
+  if (!pseudo1D_) {
```

Comment:
Try to make default normal behavior. This is less confusing than using negative statements. https://schneide.blog/2014/08/03/dont-ever-not-avoid-negative-logic/